ajax/libs/peaks.js/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "peaks.js",
+  "version": "0.6.0",
+  "description": "Frontend app for displaying audio waveforms",
+  "homepage": "http://waveform.prototyping.bbc.co.uk/",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/bbc/peaks.js.git"
+  },
+  "author": "BBC <appadmin@bbc.co.uk> (http://www.bbc.co.uk/opensource/)",
+  "license": "LGPL-3.0",
+  "filename": "main.min.js",
+  "keywords": [
+    "audio",
+    "visualisation",
+    "bbc",
+    "webaudio",
+    "browser",
+    "interactive",
+    "waveform"
+  ],
+  "npmName": "peaks.js",
+  "npmFileMap": [
+    {
+      "basePath": "src",
```

Comment:
We should fetch `peaks.js` and `peaks.js.map` under root of peaks.js npm package.